requirements.txt

```diff
@@ -2,30 +2,30 @@ cachetools==2.1.0
 coincurve==11.0.0
 colorama==0.3.9
 eth-keyfile==0.5.1
-eth-keys==0.2.1
-eth-utils==1.4.1
-filelock==3.0.8
-Flask-Cors==3.0.6
-Flask-RESTful==0.3.6
+eth-keys==0.2.2
+eth-utils==1.5.2
```

Comment:
@karlb upgrading `eth-utils` did not introduce any type checking problems in the client, is it a problem with the services?

Reply:
Did you really install eth-utils 1.5.2? I don't see any changes in the `constraints.txt`.

Reply:
Yes, this needs updates in the constraints files: `Collecting eth-utils==1.4.1 (from -c constraints.txt (line 22))`

Reply:
I thought the constraints.txt was not used anymore, let me update it

Reply:
okay, so I added an script that checks for duplicated dependencies and removed the `eth-utils` from the constraints.txt, let's see if it breaks.

Reply:
yeah, the upgrade did indeed work

Reply:
Great! There was a large mypy merge since I check it last week. That probably explains why it worked without errors this time. But I'm still surprised there's no error due to conflicting `Address` definitions.

Reply:
`eth-utils` is still at 1.4.1 for me. And there are still conflicting versions given in `requirements.txt` and `constraints.txt`.